producer.py

```diff
@@ -1,7 +1,5 @@
+from robocorp import http, workitems
 from robocorp.tasks import task
-from robocorp import http
-from robocorp import workitems
-
 from RPA.JSON import JSON
```

Comment:
I'm a bit on the fence if we want to use this library just to load a json file. It's two lines in the standard library.

Reply:
I am fine either way. It is only used for the download in the courses and it is the way we want to go forward. So it is good enough for the courses even if very incomplete.

Reply:
The reason was that we want to guide our users to first use our libraries and if we have nothing to go searching in the outside world.